src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -24,20 +27,18 @@ static Https_ClientCredentialTypeTests()
 #if FULLXUNIT_NOTSUPPORTED
     [Fact]
 #else
-    [ConditionalFact(nameof(Root_Certificate_Installed), nameof(Basic_Authentication_Available))]
+    [ConditionalFact(nameof(Root_Certificate_Installed))]
 #endif
     [OuterLoop]
     public static void BasicAuthentication_RoundTrips_Echo()
```

Comment:
Not required for this PR, but we need some negative tests, especially now that we can control the server-side authentication.  Would you mind opening a new issue to add negative tests for BasicAuth?